Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.13.15-alpine AS builder
+
+RUN apk add --update --no-cache ca-certificates bash make gcc musl-dev git openssh wget curl
+
+WORKDIR /go/src/kubepug
+
+COPY go.mod .
```

Comment:
Can we make the whole copy as one line? This would generate less layers. Like COPY . /go/src/kubepug (as an example).